### PR TITLE
breaking change: Convert channel members to map (instead of Set)

### PR DESCRIPTION
**Pull request by proggen-com**
dart fmt on all code
move onMemberAdded callback below adding member to channel

### CODE REVIEW

File: lib/pusher-js/core/options.dart [hunk 1]
```diff
@@ -80,7 +80,7 @@ abstract class Options {
       num httpPort,
       num httpsPort,
       bool ignoreNullOrigin,
-  //    nacl nacl,
+      //    nacl nacl,
```
Comment (fbenevides) [inline lib/pusher-js/core/options.dart:83]: What do you think about removing this comment?

File: lib/pusher_channels_flutter.dart [hunk 2]
```diff
@@ -195,12 +195,13 @@ class PusherChannelsFlutter {
...
       case 'onMemberAdded':
         var member = PusherMember(user["userId"], user["userInfo"]);
+        channels[channelName]?.members[member.userId] = member;
         onMemberAdded?.call(channelName!, member);
-        channels[channelName]?.members.add(member);
         channels[channelName]?.onMemberAdded?.call(member);
         return Future.value(null);
       case 'onMemberRemoved':
         var member = PusherMember(user["userId"], user["userInfo"]);
+        channels[channelName]?.members.remove(member.userId);
```
Comment (JimmyTai) [inline lib/pusher_channels_flutter.dart:204]: Does here need a null check ? before remove method?

Comment (jimbofoo) [reply lib/pusher_channels_flutter.dart:204]: Yes either that or a !, but that could theoretically cause an exception if the channel could not be found.

File: lib/pusher-js/core/options.dart (hunk 1 above)
Comment (proggen-com) [reply lib/pusher-js/core/options.dart:83]: done

**Review by fbenevides — APPROVED** [review]

**Pull request merged**